src/lib/address/address.ts

```diff
@@ -116,11 +116,13 @@ export const parseMaybeUrl = async ({ value }: { value: string }): Promise<Parse
         }
       }
     } catch (error) {
-      // Log the error and continue iterating over the remaining supportedChainIds
-      moduleLogger.trace(error, `Failed to validate address for chainId: ${chainId}`)
+      // Error validating the current ChainId, not an actual error but the normal flow as we exhaust ChainIds parsing.
```

Comment:
doesn't seem like anything in this flow actually throws, but doesn't hurt in case something is added that does throw at some point

Reply:
Double-checked this, indeed we swallow all errors down the line, but agreed to keep for safety